docs/diagrams/DeleteSequenceDiagram.puml

```diff
@@ -13,42 +14,48 @@ box Model MODEL_COLOR_T1
 participant ":Model" as Model MODEL_COLOR
 end box
 
-[-> LogicManager : execute("delete 1")
+[-> LogicManager : execute("delete c/CS2103T")
 activate LogicManager
 
-LogicManager -> AddressBookParser : parseCommand("delete 1")
-activate AddressBookParser
+LogicManager -> GradPadParser : parseCommand("delete c/CS2103T")
+activate GradPadParser
```

Comment:
I think our current delete system still works with indexes, but possibly can KIV for future updates?

Reply:
Yup for now it still works with indexes but I think we are planning to change it to ModuleCode in the near future so I just change the documentation first.